### DISCUSSION TITLE
Does Qfield support SQLite trigger on SpatiaLite layers ?

### DISCUSSION THREAD

**Discussion by Ipickedausername** [opengisch/QField#5223] [Q&A]
Hi,

Before turn my whole project into SpatiaLite, I would like to know if these layers are working properly on Qfield and if I can expect triggers UPDATE / DELETE to work ? 

Thank you !
Bye

**Comment by nirvn** [comment]
@Ipickedausername , on cable exports (when layer isn't offlined) or self made projects, it'll surely work. 

I'd love to see you doing a simple test as to whether it'd work through the cloud when offlining layers there. It won't be triggered while opening the cloud project on QField itself but has a fair chance of working when the cloud server applies the change deltas.

**Reply by Ipickedausername** [reply]
I'm mostly using offline layers. So I guess it will not work the way I expect. 

I need to update a parent layer, once the child layer has been modified. An update trigger would do the work. 

I can do some tests as soon as I have a moment.

**Reply by nirvn** [reply]
@Ipickedausername , with offline layers it would (likely) work when synchronizing back. It just requires a confirmation via testing, hoping you can bring us some good news on that front :)
